### ISSUE TITLE
Fix focus ring in Alert X button

### ISSUE THREAD

**Issue by radoslavzeman** [bratislava/bratislava.sk#1907]
Alert close button does not have proper focus ring:

<img width="1340" height="507" alt="Image" src="https://github.com/user-attachments/assets/0596911c-b640-4a1d-8969-497f82697a09" />
